src/components/SingleName/SaveCancel.js

```diff
@@ -24,33 +24,30 @@ const ActionButton = ({
   isValid
 }) => {
   // Ignore isValid == undefined
+  const { toggleModal } = useContext(GlobalState)
```

Comment:
You should add this line above the comment `// Ignore isValid == undefined`

Reply:
done